### PR TITLE
remove the word "support" from Windows pages, add Windows Server 2019

### PR DESCRIPTION
**Pull request by acozine**
##### SUMMARY
The list of Windows OSs you can manage with Ansible was different on the Windows FAQ and Windows Setup pages. Also, we used the word "support" on these pages in a confusing way. This PR updates the list of Windows OSs Ansible can manage and makes it clear that the lists reflect Ansible's technical capability, not support options from Red Hat.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
docs.ansible.com
windows modules

### CODE REVIEW

File: docs/docsite/rst/user_guide/windows_setup.rst [hunk 1]
```diff
@@ -313,8 +313,8 @@ options are:
 * ``Service\Auth\CbtHardeningLevel``: Specifies whether channel binding tokens are
   not verified (None), verified but not required (Relaxed), or verified and
   required (Strict). CBT is only used when connecting with NTLM or Kerberos
-  over HTTPS. The downstream libraries that Ansible currently uses only support
-  passing the CBT with NTLM authentication. Using Kerberos with
+  over HTTPS. The downstream libraries that Ansible currently uses can only
+  pass the CBT with NTLM authentication. Using Kerberos with
```
Comment (jborean93) [inline docs/docsite/rst/user_guide/windows_setup.rst]: NTLM or Kerberos authentication

Comment (acozine) [reply docs/docsite/rst/user_guide/windows_setup.rst]: Is this still true? 

```
Using Kerberos with
  ``CbtHardeningLevel = Strict`` will result in a ``404`` error.
```

Comment (jborean93) [reply docs/docsite/rst/user_guide/windows_setup.rst]: Nope Kerberos now supports passing the CBT as well as NTLM, sorry should have mentioned that before.

**Comment by jborean93** [comment]
@acozine I just removed that whole entry around CBT is only supported by Ansible on NTLM and Kerberos. It's not needed as they are the only 2 ones that actually use it which is already mentioned above. The entry was originally there because we only worked with CBT if using NTLM but that is no longer the case.

**Pull request merged**